cdk/opensearch-website-search/lambdas/indexer-lambda/doc-indexer.py

```diff
@@ -250,13 +250,15 @@ def do_indexing(os_client, user_params):
                                  request_timeout=20)
     print("Existing alias - Bulk response: ", bulk_response)
     print(SECTION_SEPARATOR)
-
+    
+    indices_to_remove = [index for index in os_client.indices.get_alias(name=INDEX_ALIAS)]
+    
+    actions=[{"remove": {"index": index, "alias": INDEX_ALIAS}} for index in indices_to_remove]
+    actions.append({"add": {"index": new_index, "alias": INDEX_ALIAS}})
+    
     alias_status = os_client.indices.update_aliases(
       {
-        "actions": [
-          {"remove": {"index": old_index, "alias": INDEX_ALIAS}},
```

Comment:
Previously, we tried to remove old index. This will cause issues with parallel deployments. When we try to remove old index, it may have already been deleted by another parallel thread. So, here ideal solution will be to remove all the indices that are present and add new index. By this we can always get number of indices mapped to index aliases as one or zero(at initial state).